pyatlan/generator/templates/referenceable_methods.jinja2

```diff
@@ -22,3 +22,14 @@
 
     def flush_custom_metadata(self):
         self.business_attributes = self._metadata_proxy.business_attributes
+
+    @property
+    def atlan_tag_names(self) -> list[str]:
+        from pyatlan.cache.atlan_tag_cache import AtlanTagCache
+
+        if self.classification_names:
+            return [
+                AtlanTagCache.get_name_for_id(tag_id) or f"{tag_id}-archived"
```

Comment:
Curious why we use `{tag_id}-archived` here as a fallback value. Didn't we use some other sentinel value for deleted elsewhere? (Should be consistent here, I think.)